SoftLayer/CLI/vlan/create.py

```diff
@@ -0,0 +1,40 @@
+"""Order/create a dedicated server."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+from SoftLayer.managers import ordering
+
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+@click.command(epilog="See 'slcli server create-options' for valid options.")
```

Comment:
```suggestion
@click.command(epilog="See 'slcli vlan create-options' for valid options.")
```
A new module will be needed (create-options).